scripts/update-homebrew.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Pushes the latest Yarn version to Homebrew
+
+set -ex
+
+version=`curl --fail https://yarnpkg.com/latest-version`
+
+# Ensure Linuxbrew is on the PATH
+PATH=$PATH:$HOME/.linuxbrew/bin/
+# Ensure homebrew-core is pointing to Homebrew rather than Linuxbrew
+pushd ~/.linuxbrew/Library/Taps/homebrew/homebrew-core
+#git remote set-url origin https://github.com/Daniel15/homebrew-core # for testing
+git remote set-url origin https://github.com/homebrew/homebrew-core
+# Remove any existing branch (eg. if the previous attempt failed)
+git branch -D yarn-$version || true
```

Comment:
this should switch to master branch (and pull) before deleting the branch, right?

EDIT: It automatically branches from origin/master (https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/bump-formula-pr.rb#L299), but deleting the branch would fail if you are on it, so switching and at least doing `git fetch --all` might be a good idea 😄